lib/xcpretty/parser.rb

```diff
@@ -193,7 +193,7 @@ module Warnings
 
       # @regex Captured groups
       # $1 = whole warning
-      GENERIC_WARNING_MATCHER = /^warning:\s(.*)$/
```

Comment:
Okay, blacklisting is a better model anyways.
How about this, I remove that generic catch, and we go with the rest of the commit?

-Rashin